src/core/dbus-unit.c

```diff
@@ -323,6 +323,40 @@ static int property_get_load_error(
         return sd_bus_message_append(reply, "(ss)", NULL, NULL);
 }
 
+static int property_get_markers(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        unsigned *markers = userdata;
+        int r;
+
+        assert(bus);
+        assert(reply);
+        assert(markers);
+
+        r = sd_bus_message_open_container(reply, 'a', "s");
+        if (r < 0)
+                return r;
+
+        /* Make sure out values fit in the bitfield. */
+        static_assert(_UNIT_MARKER_MAX <= sizeof(((Unit){}).markers) * 8,
```

Comment:
assert_cc() for compat with pre-C11 compilers

Reply:
Pfff, yeah, I somehow forgot we use assert_cc() for that. Fixed now.